Doc/library/ensurepip.rst

```diff
@@ -92,7 +93,7 @@ Module API
    Returns a string specifying the available version of pip that will be
    installed when bootstrapping an environment.
 
-.. function:: bootstrap(root=None, upgrade=False, user=False, \
+.. function:: bootstrap(root=None, prefix=None, upgrade=False, user=False, \
                         altinstall=False, default_pip=False, \
                         verbosity=0)
```

Comment:
Maybe add `prefix` at the end? I can't imagine many people supply arguments to this function positionally, but why break code if we don't need to?